### PR TITLE
Dev iacampo update readme

### PR DESCRIPTION
**Pull request by a-iacampo**
### Description
- Added Description, Objective and Features (core & extra)

### CODE REVIEW

**Review by Eltom777 — APPROVED** [review]
Good job !

File: README.md [hunk 1]
```diff
@@ -1,14 +1,28 @@
 # SOEN341-PicturePoint
 
 ## Description
+PicturePoint is a photo-sharing web application. Similar to [Instagram](https://help.instagram.com/424737657584573), users may upload pictures to their profile, follow other users' and like/comment on users' published picutres. This application lets PicturePointers create and be part of groups. Members within these groups will be able to upload and download pictures from this shared space. 
 ## Objective
+PicturePoint focuses on providing a platform to share pictures within a group. This application can be viewed as a shared cloud storage space for pictures. Each storage space will be controlled by a group of users.
```
Comment (digathomas) [inline README.md:6]: The privacy of these shared pictures can also be configured by it's creator.

**Comment by digathomas** [comment]
Well-worded.

File: README.md (hunk 1 above)
Comment (a-iacampo) [reply README.md:6]: Committed change to the master

**Pull request merged**